tests/e2e/routing.spec.ts

```diff
@@ -9,10 +9,13 @@ const expectIndexPage = async (
     selectedCountry: string,
     scale: string,
     featureCount: number,
-    selectedCountryFeatureCount: number
+    selectedCountryFeatureCount: number,
+    selectedCountryName: string
 ) => {
     await page.waitForURL(new RegExp(`${BASE_URL}${url}`));
-    await expect(await page.textContent(".indicator-menu-activator")).toBe(selectedIndicatorName);
+    await expect(await page.textContent(".indicator-menu-activator-desktop")).toBe(
+        `${selectedCountryName} | ${selectedIndicatorName}${selectedCountry ? " | Admin 2" : ""}`
+    );
```

Comment:
Maybe worth converting to a test that is agnostic as to which pipe character is used:

```suggestion
    await expect(await page.textContent(".indicator-menu-activator-desktop")).toContainText(selectedCountryName)
    await expect(await page.textContent(".indicator-menu-activator-desktop")).toContainText(selectedIndicatorName)
    if (selectedCountry) {
      await expect(await page.textContent(".indicator-menu-activator-desktop")).toContainText("Admin 2")
    } else {
      await expect(await page.textContent(".indicator-menu-activator-desktop")).not.toContainText("Admin")
    }
    );
```